Gruntfile.js

```diff
@@ -166,6 +166,14 @@ module.exports = function (grunt) {
       }
     },
 
+    // ESLint is a replacement for JSHint + includes style linting + is extensible
+    eslint: {
+        target: [
+          'Gruntfile.js',
+          '<%= yeoman.app %>/scripts/{,*/}*.js'
+        ]
+    },
```

Comment:
@spadgett After some digging, I found that this is actually something you can configure in the `.eslint.yml` file, for example:

```yml
eqeqeq: error
no-alert: error
no-console: 'warn'
no-tabs: 'off'
other-pretend-rule:
  - 'can-support-list'
  - 'more-than-one-way-approved'
```

Running it:

```bash
origin-web-console/app/scripts/services/data.js
   100:64  warn  'current' is defined but never used         no-unused-vars
   167:59  warn  'statusText' is defined but never used      no-unused-vars
   201:59  warn  'statusText' is defined but never used      no-unused-vars
   270:59  warn  'statusText' is defined but never used      no-unused-vars
   370:9   warn  'force' is assigned a value but never used  no-unused-vars
   392:61  warn  'statusText' is defined but never used      no-unused-vars
   932:61  warn  'statusText' is defined but never used      no-unused-vars
   958:61  warn  'statusText' is defined but never used      no-unused-vars
   982:7   error  Unexpected console statement                no-console
  1068:76  warn  'event' is defined but never used           no-unused-vars
  1079:26  error  Expected '===' and instead saw '=='         eqeqeq
```

This could be nice if we want to initially flip-on a rule to "warn", and then eventually change it to "error" once we have let it settle a bit.